servicex_app/servicex_app/resources/internal/transformer_file_complete.py

```diff
@@ -71,57 +115,55 @@ def put(self, request_id):
         return "Ok"
 
     @staticmethod
-    @retry(stop=stop_after_attempt(3),
-           wait=wait_exponential_jitter(initial=0.1, max=30),
-           before_sleep=before_sleep_log(current_app.logger, logging.INFO),
-           after=after_log(current_app.logger, logging.INFO),
-           )
-    def record_file_complete(logger: Logger, request_id: str, info: dict[str, str]) -> TransformRequest | None:
-        transform_req = TransformRequest.lookup(request_id)
-        if transform_req is None:
-            msg = f"Request not found with id: '{request_id}'"
-            logger.error(msg, extra={'requestId': request_id})
-            return None
+    @file_complete_ops_retry
+    def record_file_complete(session: Session, logger: Logger, request_id: str,
+                             info: dict[str, str]) -> TransformRequest | None:
 
-        if info['status'] == 'success':
-            TransformRequest.file_transformed_successfully(request_id)
```

Comment:
Do these methods (`file_transformed_successfully`, `file_transformed_unsuccessfully`) still exist on `TransformRequest`? If so, should they be removed?

Reply:
Ah, good call.. Removed those methods from TransformRequest